src/MergeContext.js

```diff
@@ -592,31 +622,9 @@ class CommitMessage
         return (label && label.includes('-')) ? label : null;
     }
 
-    // basic checks for an unparsed message
-    _checkRaw(message) {
-        // CRs in CRLF sequences already removed
-        // other CRs are not treated specially (and are banned)
-        const lines = message.split('\n');
-        for (let i = 0; i < lines.length; ++i) {
-            const line = lines[i];
-            const invalidPosition = this._invalidCharacterPosition(line);
-
-            if (invalidPosition !== -1)
-                throw new Error(`bad character at line ${i}, offset ${invalidPosition}`);
-
-            // allow excessively long whitespace-only lines
-            // that some copy-pasted PR descriptions may include
-            const trimmedLine = this._trim(line);
-
-            // TODO: Allow longer header (and possibly even trailer) lines.
-            if (trimmedLine.length > 72)
-                throw new Error(`too long line '${trimmedLine}'`);
-        }
-    }
-
     _parse(prDescriptionRaw) {
-        const prDescription = prDescriptionRaw.replace(/\r+\n/g, '\n');
-        this._checkRaw(prDescription);
+        const prDescriptionUntrimmed = prDescriptionRaw.replace(/\r+\n/g, '\n');
```

Comment:
I think this replacement should be moved to _parseRawLines() now. There is no reason to expose that specific step to _parse().

Reply:
Done.